internal/appv2/patches.go

```diff
@@ -243,6 +244,18 @@ func ensureArrayOfMap(raw any) ([]map[string]any, error) {
 	return out, nil
 }
 
+func patchMounts(cfg map[string]any) (map[string]any, error) {
+
+	if mount, ok := cfg["mount"]; ok {
```

Comment:
what's the behavior if `[mount]` and `[mounts]` are both defined?

Reply:
`mount` overrides `mounts` and IMO that is fine, very weird to see both on same file.

Reply:
yeah, probably not a valid config file if both are defined. what if we errored in that case that way the developers has to update the config file to use only one?

Reply:
@tvdfly i could easily add that validation. but, I was [following a convo](https://flyio.discourse.team/t/fly-toml-mounts-v-mount/1905) you were having with Michael on discourse and it turns out we accept both the singular and plural form for all the sections. and so if we are going to do it for **mount**, then we might as well have the validation on all the sections?

i think we can just leave it as is. what do you think?

Reply:
oh gotcha. it looks like we're overwriting checks with check when both are specified in the nomad platform, so that seems like a good choice here.